### PR TITLE
Remove details about message passing optimization

### PR DESCRIPTION
**Pull request by sneakerkg**
## Description
Remove details about message passing optimization.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [x] Related issue is referred in this PR

### CODE REVIEW

**Comment by zhjwy9343** [comment]
I am OK with this modification.

File: docs/source/guide/message-efficient.rst [hunk 1]
```diff
@@ -6,41 +6,12 @@
...
-   optimization)
-
-According to the above, a common practise to leverage those
+passing. A common practise to leverage those
 optimizations is to construct one's own message passing functionality as
 a combination of :meth:`~dgl.DGLGraph.update_all` calls with built-in
 functions as parameters.
 
-For some cases like
+Besides that, considering the number of edges is much larger than the number of nodes for some graphs, avoiding unnecessary memory copy from nodes to edges is beneficial. For some cases like
```
Comment (mufeili) [inline docs/source/guide/message-efficient.rst:14]: "considering" -> "considering that"

Comment (sneakerkg) [reply docs/source/guide/message-efficient.rst]: Updated.

**Review by mufeili — APPROVED** [review]
LGTM.

**Pull request merged**